### ISSUE TITLE
The ecosystem dropdown menu is inaccessible with screen readers

### ISSUE THREAD

**Issue by JacksonChen666** [matrix-org/matrix.org#1751]
Note: Related to [Zola rewrite website](https://matrix.turbo.fish/)

Tested with macOS Voiceover.

The "Ecosystem" drop down menu only shows on hover which is a problem for screen readers (or specifically macOS Voiceover), as the links within the "Ecosystem" drop down cannot be navigated to with a screen reader because it doesn't seem to trigger `:hover`.

<details><summary>"Transcript"</summary>

> "Ecosystem, group"

Read next word (Voiceover modifier+Right arrow key, name seems misleading)

> "Ecosystem , clickable"

(Voiceover modifier+space)

> *Select sound*, nothing happens

(Voiceover modifier+space)

> *Select sound*, nothing happens

(Voiceover modifier+space)

> *Select sound*, nothing happens

(Voiceover modifier+space)

> *Select sound*, nothing happens

Read next word (Voiceover modifier+Right arrow key)

> "end of, Ecosystem, group"

</details>

**Comment by thibaultamartin** [comment]
Very valuable thanks a lot for testing this! This must definitely be fixed prior to launch

**Comment by thibaultamartin** [comment]
Closing as duplicate of #1749. The issue is tracked and will be fixed before launch
